package.json

```diff
@@ -19,6 +19,9 @@
     "@vue/cli-service": "~5.0.0",
     "eslint": "^7.32.0",
     "eslint-plugin-vue": "^8.0.3",
+    "sass": "^1.49.9",
+    "sass-loader": "^10.2.1",
+    "vue-svg-loader": "^0.16.0",
```

Comment:
vue-svg-loader is a nice tool - it allows you to use svg's as components. Very cool that you used this! I just wanted to make sure if you know that it is also possible not to use this loader and use .svg files? when you import .svg (without this loader) the file loader is used and imported thing is just a url for the image asset so you can use it with e.g. <img>:

```js
import logo from  '@/assets/logo.svg'
console.log(logo) // =>  /img/logo.064c6128.svg
```